peachjam/admin.py

```diff
@@ -115,13 +116,50 @@ class DocumentTopicInline(admin.TabularInline):
     extra = 1
 
 
+class DateSelectorWidget(forms.MultiWidget):
+    def __init__(self, attrs=None):
+        months = [x for x in enumerate(list(calendar.month_name))][1:]
```

Comment:
Can you check how SelectDateWidget handles the month names, because ideally they need to be translated.

Reply:
It uses the months list from `django.utils.date` module which wraps the names in translator tags. Very useful.